<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QComboBoxDemo</class>
 <widget class="QWidget" name="QComboBoxDemo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QComboBoxDemo</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_1">
       <property name="text">
        <string>姓名：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_Name"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>学校：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbbox_School">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>清华大学</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>北京大学</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>浙江大学</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>复旦大学</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>上海交通大学</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>南京大学</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>西安交通大学</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>中国科学技术大学</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>哈尔滨工业大学</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>省份：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbbox_Province">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>城市：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbbox_City">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>结果：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_Result">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
